src/NServiceBus.Core/Transports/Msmq/QueuePermissions.cs

```diff
@@ -59,8 +59,6 @@ public static void SetPermissionsForQueue(MessageQueue queue, string account)
         {
 
             queue.SetPermissions(LocalAdministratorsGroupName, MessageQueueAccessRights.FullControl, AccessControlEntryType.Allow);
-            queue.SetPermissions(LocalEveryoneGroupName, MessageQueueAccessRights.WriteMessage, AccessControlEntryType.Allow);
```

Comment:
I looked at the history and had to traverse some renames :-)

In 7c469a98a12dc5637e628e46bc57adac66c1ee18 Anonymous was added and in f6e5bda526625545598d4c07593b4b02ea0bd1c5 Everyone. Both commits do not reference an issue and are from ~ November 2009 and made by @mattburton so a long long time ago.